benches/bigint.rs

```diff
@@ -73,7 +73,16 @@ fn u512_mul(b: &mut Bencher) {
 	b.iter(|| {
 		let n = black_box(10000);
 		let one = black_box(U512::one());
-		(1..n).fold(one, |old, new| { old.overflowing_mul(U512::from(black_box(new))).0 })
+		(1..n).fold(one, |old, new| { old.overflowing_mul(U512::from(black_box(new | 1))).0 })
+	});
+}
+
+#[bench]
+fn u512_mul_small(b: &mut Bencher) {
+	b.iter(|| {
+		let n = black_box(10000);
+		let one = black_box(U512::one());
+		(1..153).fold(one, |old, new| { old.overflowing_mul(U512::from(10)).0 })
```

Comment:
Ah sorry, I didn't commit my fixups to this (`n` is unused too). Sorry, that was a mistake. Also, 10 just because it's a non-power-of-two number and because my monkey brain likes the number 10. It could be anything, as long as it's less than 2^64.